zerver/context_processors.py

```diff
@@ -86,7 +86,7 @@ def zulip_default_context(request: HttpRequest) -> Dict[str, Any]:
 
     user_is_authenticated = False
     if hasattr(request, 'user') and hasattr(request.user, 'is_authenticated'):
-        user_is_authenticated = request.user.is_authenticated.value
+        user_is_authenticated = request.user.is_authenticated
```

Comment:
Documented in the commit msg,
```
 is_authenticated no longer returns a callable:
  code.djangoproject.com/ticket/25847
```

Reply:
ok -- could we merge this on 1.11 with its own commit message, or was the compatibility situation not correct there?

Reply:
Based on reading the ticket, a proper description should be `is_authenticated is a callable property instead of just a method`. If the ticket applies only to 2.0, then it is dangerous to do `request.user.is_authenticated` (without `()`) on 1.11.

Reply:
Doing some research, it's clear that is_authenticated is a CallableProperty in 1.11 and in 2.x is just a Property.  In other words, `request.user.is_authenticated` is fine in all versions; so we should just switch to that.  One can confirm from the fact that Zulip already uses `request.user.is_authenticated` in like 7 other places.

Reply:
Merged a3f08f01ec0113e5731c0a3c483a7ed54a4a1be8 as this was correct to do on 1.11.